simuvex/procedures/syscalls/mmap.py

```diff
@@ -82,10 +82,6 @@ def run(self, addr, length, prot, flags, fd, offset): #pylint:disable=arguments-
             # Can't give you that address. Find a different one.
             addr = self.allocate_memory(size)
```

Comment:
self.allocate_memory only returns the address of the memory to allocate. If the map_region fails because of an already present allocation we will still get fully symbolic data I think.

The same issue happens above in the case of addr being zero